<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorActorParameters</class>
 <widget class="QWidget" name="EditorActorParameters">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QPushButton" name="button_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>498</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="button_ok">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="EditorActorParameterPage" name="page_maxhp">
      <attribute name="title">
       <string>MaxHP</string>
      </attribute>
     </widget>
     <widget class="EditorActorParameterPage" name="page_maxsp">
      <attribute name="title">
       <string>MaxSP</string>
      </attribute>
     </widget>
     <widget class="EditorActorParameterPage" name="page_str">
      <attribute name="title">
       <string>STR</string>
      </attribute>
     </widget>
     <widget class="EditorActorParameterPage" name="page_dex">
      <attribute name="title">
       <string>DEX</string>
      </attribute>
     </widget>
     <widget class="EditorActorParameterPage" name="page_agi">
      <attribute name="title">
       <string>AGI</string>
      </attribute>
     </widget>
     <widget class="EditorActorParameterPage" name="page_int">
      <attribute name="title">
       <string>INT</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditorActorParameterPage</class>
   <extends>QWidget</extends>
   <header>editors/actors/editoractorparameterpage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>button_ok</tabstop>
  <tabstop>button_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
